mne/viz/circle.py

```diff
@@ -119,7 +119,6 @@ def _plot_connectivity_circle_onpick(event, fig=None, axes=None, indices=None,
         fig.canvas.draw()
 
 
-@deprecated(CONNECTIVITY_DEPRECATION_MSG)
 def plot_connectivity_circle(con, node_names, indices=None, n_lines=None,
```

Comment:
@adam2392 @agramfort I think we need to un-deprecate this because `plot_channel_labels_circle` calls `plot_connectivity_circle` under the hood.

Another option would be to still deprecate `plot_connectivity_circle` but make it private in MNE-Python. That might be slightly cleaner, not sure. Thoughts on which is better?

Reply:
I'm okay with un-deprecate it and move it back into MNE-python if needed.